service/controller/v12/resource/instance/create.go

```diff
@@ -601,7 +612,13 @@ func firstInstanceInProgress(customObject providerv1alpha1.AzureConfig, list []c
 // bundle version of the custom object and the current version bundle version of
 // the instance's tags applied. In case all instances are reimaged
 // firstInstanceToReimage return nil.
-func firstInstanceToReimage(customObject providerv1alpha1.AzureConfig, list []compute.VirtualMachineScaleSetVM, instanceNameFunc func(customObject providerv1alpha1.AzureConfig, instanceID string) (string, error), versionValue map[string]string) (*compute.VirtualMachineScaleSetVM, error) {
+func firstInstanceToReimage(
+	customObject providerv1alpha1.AzureConfig,
+	list []compute.VirtualMachineScaleSetVM,
+	instanceNameFunc func(customObject providerv1alpha1.AzureConfig, instanceID string) (string, error),
+	versionValue map[string]string,
+	desiredDiskSizes map[string]int32,
```

Comment:
We don't usually pass this kind of information as separate parameter, but extract it with key function from `customObject`.

Reply:
removed